pkg/cmd/pulumi/org.go

```diff
@@ -64,6 +65,9 @@ func newOrgCmd() *cobra.Command {
 
 	cmd.AddCommand(newOrgSetDefaultCmd())
 	cmd.AddCommand(newOrgGetDefaultCmd())
+	if env.Dev.Value() {
+		cmd.AddCommand(newSearchCmd())
+	}
```

Comment:
I assume we want to remove this before shipping?

Reply:
Plan is to initially ship requiring the envvar for now?

Reply:
> Plan is to initially ship requiring the envvar for now?

Works for me! Was just curious.